include/ocpp/v201/charge_point.hpp

```diff
@@ -722,6 +739,12 @@ class ChargePoint : ocpp::ChargingStationBase {
                                                 const AttributeEnum& attribute_enum) {
         return this->device_model->request_value<T>(component_id, variable_id, attribute_enum);
     }
+
+    /// @brief Switch to a specifc network connection profile given the configuration slot.
```

Comment:
I would say try to switch here, or even do nothing. It depends, if this is called multiple times and every time the switch is not possible, then ignore it for example. Or when the network connection profile priority is lower, this can be ignored as well I think.